code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -9,6 +9,7 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
  * as well as allowing users to save and load configurations.
  */
 /datum/nttc_configuration
+	var/regex/word_blacklist = new("(<iframe|<embed|<script|<svg|<canvas|<video|<audio|onload)") // Blacklist of naughties
```

Comment:
Does this check for capitalisation later on? Otherwise you need to do that

Reply:
90% sure, testing now